src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -302,8 +302,10 @@ private void OnPreRender()
             }
         }
 
-        private void OnPostRender()
+        private IEnumerator OnPostRender()
         {
+            yield return new WaitForEndOfFrame();

```

Comment:
>yield return new WaitForEndOfFrame(); [](start = 12, length = 37)

Can you add a comment about why this is here? It seems like it's to wait for other scripts that have OnPostRender steps to finish executing first